crates/common/tedge_config/src/tedge_config_cli/tedge_config.rs

```diff
@@ -436,7 +436,7 @@ define_tedge_config! {
 
         /// Set of MQTT topics the AWS IoT mapper should subscribe to
         #[tedge_config(example = "tedge/measurements,tedge/measurements/+")]
```

Comment:
I'd suggest to update the example as well.


```suggestion
        #[tedge_config(example = "te/device/+/+/+/m/+")]
```

Reply:
fixed